index.html

```diff
@@ -693,14 +706,105 @@ <h4>Proof Verification (ecdsa-2019)</h4>
       <p>
-The following section describes security considerations that developers
-implementing this specification should be aware of in order to create secure
-software.
+The security (integrity/authenticity) of a verifiable credential 
+signed by a digital signature algorithm is dependent on a number of factors 
+including:
       </p>
-
+      <ul>
+        <li>
+the correct application of the signature algorithm to a verifiable
+credential (this specification)
+        </li>
+        <li>
+the choice of of signature algorithm (ECDSA) and its parameters (P-256, P-384)
+        </li>
+        <li>
+the correct implementation and usage of the signature algorithm 
+particularly with respect to well known problem areas
+        </li>
+        <li>
+the proper management of the private and public keys used for 
+signing and verification
+        </li>
+      </ul>
+      <p>
+In the following sections we review these important points and point 
+the readers to where to find additional information.
+      </p>
+      <section class="informative">
+        <h3>Choice of ECDSA and Parameters</h3>
+        <p>
+The ECDSA signature scheme has the <strong>EUF-CMA</strong> 
+(<em>existential unforgeability under chosen message attacks</em>) security 
+property. This property guarantees that any efficient adversary who has the 
+public key <em>pk</em> of the signer and received an arbitrary number of 
+signatures on messages of its choice (in an adaptive manner) cannot output a 
+valid signature for a new message (except with negligible probability).
+        </p>
+        <p>
+<strong>SUF-CMA</strong> (<em>strong unforgeability under chosen 
+message attacks</em>) is a stronger notion than <em>EUF-CMA</em>. It guarantees 
+that for any efficient adversary who has the public key <em>pk</em> of the 
+signer and received an arbitrary number of signatures on messages of its 
+choice, it cannot output a new valid signature pair for either a new message 
+or a new signature for an old message (except with negligible probability). 
+ ECDSA signature scheme does <strong>not</strong> have the SUF-CMA property, 
+ while other schemes such as EdDSA [[FIPS-186-5]] do.
+        </p>
+        <p>
+Per [[NIST-SP-800-57-Part-1]] in the absence of large scale quantum 
+computers a <em>security strength</em> level of 128 bits requires a key size 
+of approximately 256 bits while a security strength level of 192 bits requires 
+a key size of 384 bits. [[NIST-SP-800-186]] recommendations includes curves 
+P-256 and P-384 at these security strength levels respectively.
+        </p>
+      </section>
+      <section class="informative">
+        <h3>Implementation Considerations for ECDSA Algorithms</h3>
+        <p>
+The ECDSA algorithm as detailed in [[FIPS-186-5]] states: &quot;A 
+new secret random number <em>k</em>, shall be generated prior to the generation 
+of each digital signature for use during the signature generation process.&quot; 
```

Comment:
If you're going to quote, you must QUOTE. In the source document, the sentence reads:

> A new secret random number <em>k</em>, 0 < <em>k</em> < <em>n</em>, <strong>shall</strong> be generated prior to the generation of each digital signature for use during the signature generation process.

Based on the above *actual* quotation, our document should have either this, _with_ the range expression (note there is a comma both following `<em>k</em>` and preceding `<strong>shall</strong>`) —
```suggestion
The ECDSA algorithm as detailed in [[FIPS-186-5]] states: &quot;A 
new secret random number <em>k</em>, 0 < <em>k</em> < <em>n</em>,
<strong>shall</strong> be generated prior to the generation 
of each digital signature for use during the signature generation process.&quot; 
```

Reply:
— or this, _without_ the range expression but with the `...` indicating an omission from within the quoted material (note there is no comma following `<em>k</em>` nor preceding `<strong>shall</strong>`) —
```suggestion
The ECDSA algorithm as detailed in [[FIPS-186-5]] states: &quot;A 
new secret random number <em>k</em> ...
<strong>shall</strong> be generated prior to the generation 
of each digital signature for use during the signature generation process.&quot; 
```